apps/gkyl_gyrokinetic.h

```diff
@@ -228,6 +228,7 @@ struct gkyl_gyrokinetic_species {
 
   enum gkyl_gkmodel_id gkmodel_id;
   double charge, mass; // Charge and mass.
+  double skip_cell_threshold; // Skip updates over cells where 0th DG component of Jf is smaller than this value. Jf is what is output in the -species_#.gkyl files
```

Comment:
I think I mentioned this elsewhere, or maybe I mentioned it in Slack, but I think this comment, and the meaning of `skip_cell_threshold` should be changed so that it doesn't involve the 0th DG coefficient and instead the cell-averaged Jf. The former requires the user to know the relationship between the 0th coefficient and the cell-av Jf (i.e. the normalization), but in general they don't. So it's best to say it'll skip cells when Jf_cell-av < skip_cell_threshold, and under the hood rescale `skip_cell_threshold` by the appropriate normalization factor.

Reply:
change comment since you changed the normalization